lib/active_stash/index.rb

```diff
@@ -1,9 +1,28 @@
 module ActiveStash
   class Index
-    attr_accessor :type, :valid_ops, :options
+    attr_accessor :type, :valid_ops, :options, :name, :unique
     attr_reader :field
 
-    RANGE_TYPES = [:timestamp, :date, :datetime, :float, :decimal, :integer]
+    # TODO: boolean supports range
+    # TODO: string and text support ranges
```

Comment:
Are these tracked anywhere else or are you planning on doing these next?

Reply:
They are just things I noticed as I went along. I'm not going to fix it in _this_ PR. Next thing is to release, and test on Healthbank then revisit the TODOs.